modules/exoplanets/exoplanetsmodule_lua.inl

```diff
@@ -419,14 +410,16 @@ void createExoplanetSystem(const std::string& starName) {
     const glm::dmat4 rotation = computeOrbitPlaneRotationMatrix(meanInclination);
     const glm::dmat3 meanOrbitPlaneRotationMatrix = static_cast<glm::dmat3>(rotation);
 
-    // 1 AU Size Comparison Ring
-    const std::string ringIdentifier = starIdentifier + "_1AU_Ring";
-    const std::string ring = "{"
-        "Identifier = '" + starIdentifier + "_1AU_Ring',"
+    bool isEnabled = module->showComparisonCircle();
+    const std::string isEnabledString = isEnabled ? "true" : "false";
```

Comment:
can be a `std::string_view` since the text assigned to it is a literal

Reply:
Hmm, but `std::string_view` cannot be concatenated with an `std::string` using the `operator+` 🤔 which is done below

Reply:
Oh right; i forgot you used the "regular" concatenation... with fmt::format it would have worked, but that makes the whole line much more unwieldy...

So nevermind!